app/steps/medicaid/intro_marital_status_indicate_spouse.rb

```diff
@@ -0,0 +1,22 @@
+module Medicaid
+  class IntroMaritalStatusIndicateSpouse < Step
+    step_attributes(
+      :member_id,
+      :spouse_id,
+      :spouse,
+    )
+
+    def valid?
```

Comment:
this should probably be added as a validator instead of overriding activemodel validations

i.e.

```
validate_presence_of :spouse, message: "Make sure you select a person"
```

Reply:
The way we have this working, we show the error on `spouse` but the validation is on `spouse_id` -- the reason for this is that each checkbox represents a `spouse_id` field, so if we returned an error on that, we'd have a list of errors.

This way, the error just shows up once. A little weird, but it's the way we've validated all of these lists of checkboxes for other steps so it seems to work ok. Does that make sense?

Reply:
Oh, I see, the error shows up on spouse. Ok, I'm mostly just worried about overriding the default valid? method. How about this?

```
module Medicaid
  class IntroMaritalStatusIndicateSpouse < Step
    step_attributes(
      :member_id,
      :spouse_id,
      :spouse,
    )
    
    validate :spouse_present

    def spouse_present
      if spouse_id.blank?
        errors.add(:spouse, "Make sure you select a person")
      end
    end

    def member
      @_member ||= Member.find(member_id)
    end
  end
end
```